metabase/metabase
# Add date input for timeline event dates

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Adds `DateInput`. Borrowed some date formatting logic from `SpecificDatesPicker`.
- Adds `date` form field
- Uses it for timeline events

How to test:
- Go to a collection. Click on the calendar icon
- If there are no timelines, click `Add an event`. Otherwise you need to navigate to a timeline first.
- Make sure it's possible to enter the date with keyboard
- Make sure it's possible to select a date with a calendar (mouse only, not accessible)

Follow ups:
- Add time & timezone selection

<img width="668" alt="Screenshot 2022-02-16 at 20 51 54" src="https://user-images.githubusercontent.com/8542534/154338839-54a4f710-afaa-42ac-8509-d53bcc1a092d.png">
